src/qt/sendcoinsdialog.cpp

```diff
@@ -193,7 +195,7 @@ void SendCoinsDialog::setModel(WalletModel *_model)
             settings.remove("nSmartFeeSliderPosition");
         }
         if (settings.value("nConfTarget").toInt() == 0)
-            ui->confTargetSelector->setCurrentIndex(getIndexForConfTarget(model->getDefaultConfirmTarget()));
+            ui->confTargetSelector->setCurrentIndex(getIndexForConfTarget(model->node().getTxConfirmTarget()));
```

Comment:
Are you sure this is correct? Not sure how `::nTxConfirmTarget` relates to `getDefaultConfirmTarget`